src/printer.js

```diff
@@ -0,0 +1,35 @@
+
+'use strict';
+
+var formatDebitTransaction = require('../src/formatter').formatDebitTransaction
+var formatCreditTransaction = require('../src/formatter').formatCreditTransaction
+var addBalanceToEachTransaction = require('../src/formatter').addBalanceToEachTransaction;
+var revertDateToString = require('../src/formatter').revertDateToString;
+
+function Printer (listOfTransactions) {
+  this.listOfTransactions = listOfTransactions
+}
+
+Printer.prototype.printOneLine = function (transaction) {
+  process.stdout.write(revertDateToString(transaction.date));
+  if (transaction.amount > 0) {
+    console.log(formatCreditTransaction(transaction));
+  } else {
+    console.log(formatDebitTransaction(transaction));
+  }
+};
+
+Printer.prototype.printHeader = function () {
```

Comment:
It may make methods like this easier to test if the console.logging is separated from the string creation.